assets/js/googlesitekit/widgets/components/WidgetContextRenderer.js

```diff
@@ -35,9 +35,17 @@ const { useSelect } = Data;
 const WidgetContextRenderer = ( props ) => {
 	const { slug, className, Header, Footer } = props;
 
-	const widgetAreas = useSelect( ( select ) => select( STORE_NAME ).getWidgetAreas( slug ) );
+	const widgetAreas = useSelect( ( select ) => {
+		if ( slug ) {
+			return	select( STORE_NAME ).getWidgetAreas( slug );
```

Comment:
You have a stray tab in here 😄 
```suggestion
			return select( STORE_NAME ).getWidgetAreas( slug );
```